packages/example/src/components/NavigationMenu/index.ts

```diff
@@ -0,0 +1 @@
+export { NavigationMenu } from './NavigationMenu';
```

Comment:
Not sure that is needed to create index file. The imports work great just with the component name. For the modern webpack it' even not necessary to mention the whole path if the folder and file has the same name. This practice with index file coming from the time when our bundler were less advance :)

Reply:
maybe that work with CRA but unfortunately it is not the case with Vite (as I know)